plugins/plugin-client-default/notebooks/dashboard.md

```diff
@@ -0,0 +1,31 @@
+---
+layout:
+  1: default
+  2: default
+  3: default
+---
+
+```bash
+---
+execute: now
+---
+ray job info raysubmit_mP8gXdLKVTkdEwfX
```

Comment:
@starpit do you think we should hardcode some random ids for this commit or you have a variable in mind that we can use

Reply:
This is a thing that I need to figure it out with @starpit in terms on how we pass dynamic variables to the markdown files if is possible

Reply:
i think we will be working in one of two modes:

1) the ui is launched in the context of a given job. in which case there is nothing to hard-code, rather we would access the current job id. e.g. passed as an env var. or simply as a command line argument.
2) the user wants to switch to some other running or previously run job. 

perhaps `ray job logs` without any arguments should use the current job.